source/configure/enable-copilot.rst

```diff
@@ -78,6 +78,43 @@ Configure a large language model (LLM) for your Copilot integration by going to
   4. Enter your API key in the **API Key** field.
   5. Specify your model name in the **Default Model** field.
 
+Custom instructions
+====================
+
+Text input here is included in the prompt for every request. Use this to give your bots extra context or instructions. For example you could list all of your organization's specific acronyms so the bot knows what you are talking about and users can ask for definitions. Or you can give it instructions like telling it to adopt a specific personality.
+
+Enable Vision (Beta)
+=====================
+
+Enabling vision enables images that are attached to posts to be sent to the upstream LLM for analysis. This requires that your upstream LLM support this features. Only available with OpenAI and OpenAI Compatable services.
+
+Copilot plugin metrics
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Metrics for the copilot plugin are exposed through the ``/plugins/mattermost-ai/metrics`` subpath under the existing Mattermost server metrics endpoint. This is controlled by the :ref:`Listen address for performance <configure/performance-monitoring-configuration-settings:listen address for performance>` configuration setting. It defaults to port ``8067``.
+
+- ``copilot_system_plugin_start_timestamp_seconds``: The time the plugin started.
+- ``copilot_system_plugin_info``: The plugin version and installation ID.
+- ``copilot_api_time_seconds``: How long to execute API.
+- ``copilot_http_requests_total``: The total number of API requests.
+- ``copilot_http_errors_total``: The total number of http API errors.
+- ``copilot_llm_requests_total``: The total number of requests to upstream LLMs.
+
+Integrations
+~~~~~~~~~~~~
+
+Currently integrations are limited to DMs with the bots. The integrations will not operate from within a regular channel.
+
+Jira
+====
+
+Issues with public JIRA instances can be fetched. No configuration is required for this integration.
```

Comment:
Is this still the case? It can't access private Jira issues if authenticated?

Reply:
Yes this is still the case.